drivers/web-worker-driver/sqljs/build.gradle

```diff
@@ -40,3 +41,8 @@ npmPublish {
 rootProject.tasks.named("kotlinNpmInstall").configure {
   dependsOn(tasks.named("assembleJsPackage"))
 }
+
+// Dokka forces us to apply the plugin to this module even though there's no code
```

Comment:
You can remove the project from inclusion in the multi-module setup with:
```gradle

apply plugin: 'org.jetbrains.dokka'
dokkaHtmlMultiModule {
  removeChildTasks([
    project(":path:to:project"),
  ])
}
```
in the root `build.gradle`.

Reply:
That was the first thing I tried but unfortunately it had no effect.

Reply:
Huh, weird.